DEV_QUICK_START.md

```diff
@@ -41,13 +41,13 @@ To build Strimzi from source the operator and Kafka code needs to be compiled in
 
 3. Now build the Docker images and push them to your repository on Docker Hub:
     
-        make all
+        make clean all
```

Comment:
I think @ppatierno found out the other day that doing `make clean all` does nto work well because it doesn't do `make clean` and `make all` but goes by the modules first. So I guess we maybe need it as two spearate steps?

Reply:
That's a good point but that was with the `release`, in this case doing `clean` and `all` together worked fine. Maybe I can put them separately just for consistency?

Reply:
Lol, for whatever reason I didn't realised this was your PR ... I thought Tom C opened it :-D ... so sorry for the weird sounding comment :-)

Reply:
Anyway with my last commit I put them separately :)